tests/acceptance/testall

```diff
@@ -474,6 +474,9 @@ export CFENGINE_TEST_OVERRIDE_WORKDIR TEMP CFENGINE_TEST_OVERRIDE_EXTENSION_LIBR
 
         TEST_END_TIME=$(unix_seconds)
 
+        # This script does not run under GAINROOT but it needs to write logs
+        $GAINROOT chmod ugo+rwxt "$WORKDIR"
```

Comment:
Great I'll leave this in then! This was figured out after many trial and error attempts, it seems that only changing the permissions *after* the test run works.